app/views/admin/responses/_form.html.erb

```diff
@@ -0,0 +1,49 @@
+<%= form_for [:admin, consultation, consultation_response], url: [:admin, consultation, consultation_response.singular_routing_symbol] do |form| %>
+  <%= render "govuk_publishing_components/components/date_input", {
```

Comment:
I think these need to be selects. I think we can use this helper method 

https://apidock.com/rails/ActionView/Helpers/DateHelper/date_select

And steal a lot of the styling from the datetime fields component 

https://github.com/alphagov/whitehall/blob/main/app/assets/stylesheets/components/_datetime-fields.scss
https://github.com/alphagov/whitehall/blob/main/app/views/components/_datetime_fields.html.erb

Reply:
Replaced date inputs with updated datetime_fields component